### ISSUE TITLE
2.4.0-beta1 release plan

### ISSUE THREAD

**Issue by trackleft** [az-digital/az_quickstart#1715]
## 2.4.0-beta2

- #1709 
- #1702
- #1708
- #1683
- #1721
- #1687
- #1592
- #1594
- #1729 
- #1726
- #1727 
- #1680 
- #1649
- #1718
- #1713
- #1717
- #1733 
- #1732 
- #1735

**Comment by joeparsons** [comment]
## Pick list

```
git cherry-pick 06f0c04ed5ccafa8e6c0d6eec9b061b033f13f2a
git cherry-pick 65bb38bb9850f913754749424a489b65468b2b12
git cherry-pick c94ed2a7bcfb6ba90508f80f1dd1d73248fa60fb
git cherry-pick d833c69853730c4f055e38d9d684be1806aa13f0
git cherry-pick 70513afbbacbb65dadf520481655b22179960699
git cherry-pick 236507515015f83236516261f3af0542e65979ac
git cherry-pick 726fee4a4fd08876ec6af8ccadd167af305492e1
git cherry-pick f00d05c639f92622d5dbba503b2947eb86ef6ada
git cherry-pick 07a56f0e7a23d3cb890c820fe1c2f9209e5f123f
git cherry-pick 479a839d528c71a194869c651fe194f8e9daac76
git cherry-pick 7da82deb5553607b12b6449be5466af98678666e
git cherry-pick ecbf16e2cf3123b9aca715446e193080288a6672
git cherry-pick daee7757b5a32f2b4e97d0757120e78d83b5d3b8
git cherry-pick 7cd678065a602626b20032a3b99e1d1ae5a9a8b2
git cherry-pick 7697dcd6ced651c9e17f8b46b1f4b61c99968aa9
git cherry-pick 9511bd98ead8a0b205fa6651403a4cf60f75d161
git cherry-pick ad92698f0504a86014290d788524b268cc1350fc
git cherry-pick ca9cceb0f8c4a1334397052052522a5249668b13
git cherry-pick 6714524f7ed8a43a6f512d47aa8f34f33972d728
```
## Release notes

`git log 2.4.0-beta1..HEAD --pretty='format:- %s' `

```
- Closes #1734: Add patch for known issue affecting field_group module issue on 9.4.x. (#1735)
- Update drupal/entity_browser requirement from 2.6 to 2.7.0 (#1732)
- Update drupal/core-recommended requirement from 9.4.1 to 9.4.2 (#1733)
- Update drupal/ctools requirement from 3.7 to 4.0.0 (#1717)
- Closes #1700: Update text on media transparent classes (#1713)
- Update drupal/cas requirement from 1.7 to 2.0.0 (#1718)
- Update drupal/easy_breadcrumb requirement from 2.0.2 to 2.0.3 (#1649)
- Closes #1651: Add grayscale background color options (#1680)
- Closes #1697: Fix JS error when Text with Background, Text on Media are not full-width (#1727)
- Closes #1716: Add overrides for claro node form layout styles. (#1726)
- Fixes #1728 Fix course PHPstan errors. (#1729)
- Closes #1593: Increase PHPStan analysis level and fix existing problems. (#1594)
- Closes #1550 Initial Intelligence Bank integration. (#1592)
- Fixes #1576 Add Course Content Type (#1687)
- Closes #1719 - Update node version in .lando file (#1721)
- Closes #1677: Update Text on Media Bottom Style (#1683)
- Fixes #1698: Fix sidebar position when full-width content in other regions (#1708)
- Closes #1701 update Lando and DDev config to enable Drupal log in form after install. (#1702)
- Closes #1707: Update Terminus and PHP versions in review site GH actions workflow. (#1709)
- #1703 2.4.x release preparations update CI and Dev tooling
- #1703 2.4.x release preparations
```

**Comment by joeparsons** [comment]
Released: https://github.com/az-digital/az_quickstart/releases/tag/2.4.0-beta2
